src/cardproduct.css

```diff
@@ -0,0 +1,88 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card-container {
+  display: flex;
+  font-style: normal;
+}
+
+.card-product {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px ;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card-product__image {
+  display: flex;
+  justify-content: center;
+}
+
+.card-product__image_size {
+  height: 134px;
+  width: 160px;
+}
+
+.card-product__name {
+  margin-top: 40px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card-product__code {
+  height: 14px;
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card-product__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  line-height: 18px;
+}
+
+.card-product__price_title {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card-product__price_money {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.card-product__buy {
+  margin-top: 16px;
+}
+
+.card-product__buy_btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 38px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  border-radius: 5px;
+  text-decoration: none;
+  background-color: #00acdc;
```

Comment:
Make CSS variables for repeatable colors. 

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)